# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1326231811 18000
#      Tue Jan 10 16:43:31 2012 -0500
# Branch stable
# Node ID 5bb929d8b4f9145d826044fb2a051ef2189c7a46
# Parent  053404893d027370ebb22f02db04c7fd83d3afc7
maint: ensure edited configure script is executable

* autogen.sh: Set mode of edited configure script to 755.

diff --git a/autogen.sh b/autogen.sh
--- a/autogen.sh
+++ b/autogen.sh
@@ -43,9 +43,11 @@ build-aux/bootstrap "$@"
 ## gcc and g++ causes trouble.  The following will make it harder (but not
 ## impossible) for users to make this mistake.
 ##
 ## FIXME -- we should really work to fix autoconf so that it prefers gfortran
 ## over g77 even when searching for a Fortran 77 compiler.
 
 echo "replacing all occurrences of g77 with gfortran in configure script..."
 
-sed 's/g77/gfortran/g' configure > configure.t && mv configure.t configure
+sed 's/g77/gfortran/g' configure > configure.t
+mv configure.t configure
+chmod 755 configure
